__tests__/api.test.js

```diff
@@ -58,4 +59,53 @@ describe("/api/getCategories", () => {
         });
       });
   });
+
+  it("error handling, expect 404 custom error if bad path", () => {
+    return request(app)
+      .get("/api/categories/funny")
+      .expect(404)
+      .then(({ body }) => {
+        expect(body.msg).toBe(
+          "Path Not Found. The correct path is /api/categories"
+        );
+      });
+  });
+});
+////////////////////// API GET REVIEW OBJECT BY ID//////////////////////////////
+
+describe("/api/getReviewObjectById", () => {
+  it("return a sinlge entity in the form of an object", () => {
+    return request(app)
+      .get("/api/reviews/1")
+      .expect(200)
+      .then(({ body }) => {
+        expect(Array.isArray(body)).toEqual(false);
+        expect(typeof body === "object").toEqual(true);
+      });
+  });
+
+  it("return correct object given correct ID", () => {
+    return request(app)
+      .get("/api/reviews/3")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.category).toEqual(expect.any(String));
+        expect(body.created_at).toEqual(expect.any(String));
+        expect(body.designer).toEqual(expect.any(String));
+        expect(body.owner).toEqual(expect.any(String));
+        expect(body.review_body).toEqual(expect.any(String));
+        expect(parseInt(body.review_id)).toBe(3);
+        expect(body.review_img_url).toEqual(expect.any(String));
+        expect(body.title).toEqual(expect.any(String));
+      });
+  });
+
+  it("return an error if id out of bounds is used", () => {
```

Comment:
It's not that the id is out of bounds - there MAY eventually be a review with an ID of 2000 added, it's just that a review with that ID is not present in the database. For example, what if the review with ID 2 was deleted, and someone went to request that endpoint - we would expect the same error saying something like "article not found" - which, incidentally, would have something other than a 400 error code.

Reply:
fixed :)